src/org/zaproxy/zap/extension/AllInOneNotes/ZapAddOn.xml

```diff
@@ -0,0 +1,20 @@
+<zapaddon>
+	<name>All In One Notes</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>A simple extension to view all notes in one pane.</description>
+	<author>David Vassallo</author>
+	<url></url>
+	<changes>
+	<![CDATA[
+	First commit.<br>
```

Comment:
While not wrong it might make more sense to users who see this in the Marketplace etc if this was "Initial release." or something like that.

Reply:
agreed! changed